src/SplitFlapDisplay.ino

```diff
@@ -12,6 +12,7 @@ JsonSettings settings = JsonSettings("config", {
     // General Settings
     {"name", JsonSetting("My Display")},
     {"mdns", JsonSetting("splitflap")},
+    {"otaPass", JsonSetting("quackquack")},
```

Comment:
Might be good to default this to an empty string and then only start the OTA server if a password is set.